frontend/src/assets/css/loader.scss

```diff
@@ -0,0 +1,82 @@
+$loader-bg-color: #ffffff80;
+$loader-color: #cceeff;
```

Comment:
Try to use colors from figma design, also we will use css variables instead of scss. Create a variables.scss file in src/assets/css folder and define them there